kpoint-back/src/main/java/ua/in/kp/controller/ProjectController.java

```diff
@@ -36,9 +36,15 @@ public ResponseEntity<Page<GetAllProjectsDto>> getAllProjects(Pageable pageable)
         return new ResponseEntity<>(projectService.getAllProjects(pageable), HttpStatus.OK);
     }
 
-    @GetMapping("/{projectId}")
-    public ResponseEntity<ProjectResponseDto> getProjectById(@PathVariable String projectId) {
+    @GetMapping("/")
```

Comment:
do we still need this method? 
if you substitute it with getProjectByURL just remove it and all connected

Reply:
It was said to leave it

Reply:
is old method work with current implementation (getById)?

maybe lets change endpoint to @GetMapping("/id/{projectId}") and @GetMapping("url/{url}")